apps/token-manager/app/src/components/HolderRow.js

```diff
@@ -33,9 +33,14 @@ class HolderRow extends React.Component {
       address,
       balance,
       groupMode,
-      tokenDecimalsBase,
       isCurrentUser,
+      maxAccountTokens,
+      tokenDecimalsBase,
     } = this.props
+
+    const oneToken = formatBalance(balance, tokenDecimalsBase) === '1'
```

Comment:
Another way to check this would be `balance.eq(tokenDecimalsBase)`, which might be nicer since there's less conversions involved?